src/MIDebugPackage/MIDebugPackage.csproj

```diff
@@ -202,6 +206,19 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
+  <ItemDefinitionGroup Condition="'$(LocalizationEnabled)'=='true'">
+    <FilesToLocalize>
+      <Languages>$(VS)</Languages>
+      <ProjectFile>$(MSBuildThisFileFullPath)</ProjectFile>
+      <HasLceComments>true</HasLceComments>
+    </FilesToLocalize>
+  </ItemDefinitionGroup>
+  <ItemGroup Condition="'$(LocalizationEnabled)'=='true'">
+    <FilesToLocalize Include="$(OutDir)OpenFolderSchema.json">
```

Comment:
If you want to merge just this part and do the rest as a subsequent PR that is fine by me. I will approve the PR.